default.properties

```diff
@@ -43,3 +43,25 @@ headerHeight=90
 # If different from default value "/header/", adapt
 # security-proxy/targets-mapping.properties accordingly
 headerUrl=/header/
+
+# Administrator email
+# Default email address used to send and receive mails in atlas, console and
+# extractorapp. See the corresponding properties files to override this email
+# address for specific needs.
+administratorEmail=georchestra@georchestra.mydomain.org
+
+# PostgreSQL database name
+# Default common PostgreSQL database for all geOrchestra modules
+pgsqlDatabase=georchestra
```

Comment:
nitpicking: `psqlDatabase` rather than `pgsqlDatabase` and also below (sounds nicer)

Reply:
As I didn't know which one to choose between psql and pgsql, I opted for the ["official" one](https://www.postgresql.org/docs/current/acronyms.html). But we may change as well to psql.

Reply:
okay, np. Let's keep it if it's official ;-)

Reply:
OK :)